src/steps/os/linux.rs

```diff
@@ -135,7 +145,7 @@ fn upgrade_arch_linux(sudo: &Option<PathBuf>, cleanup: bool, run_type: RunType,
             .arg("--pacman")
             .arg(pacman)
             .arg("-Syu")
-            .arg("--devel")
+            .arg(yay_arguments)
```

Comment:
I don't think this will work when multiple arguments are given. We'll have to split it by space and pass it to `.args`. See [this example](https://github.com/r-darwish/topgrade/blob/ab318905a116d26e1c0ac6c86ef496cb94c8a8f3/src/steps/git.rs#L116)